alpha/alpha-fsm-channel-kafka/src/main/java/org/apache/servicecomb/pack/alpha/fsm/channel/kafka/KafkaMessageListener.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.servicecomb.pack.alpha.fsm.channel.kafka;
+
+import org.apache.servicecomb.pack.alpha.core.fsm.event.base.BaseEvent;
+import org.apache.servicecomb.pack.alpha.core.fsm.sink.ActorEventSink;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.kafka.annotation.KafkaListener;
+
+public class KafkaMessageListener {
+
+    private static final Logger logger = LoggerFactory.getLogger(KafkaMessageListener.class);
+
+    private ActorEventSink actorEventSink;
+
+    public KafkaMessageListener(ActorEventSink actorEventSink) {
+        this.actorEventSink = actorEventSink;
+    }
+
+    @KafkaListener(topics = "${alpha.feature.akka.channel.kafka.topic:servicecomb-pack-actor-event}")
+    public void listener(BaseEvent baseEvent){
+        if(logger.isDebugEnabled()){
+            logger.debug("listener event = [{}]", baseEvent);
+        }
+
+        try {
+            actorEventSink.send(baseEvent);
+        }catch (Exception e){
+            logger.error("subscriber Exception = [{}]", e.getMessage(), e);
```

Comment:
I recommend using commit the offset manually, what do you think?

Set the enable-auto-commit property to false:

```java
propsMap.put(ConsumerConfig.ENABLE_AUTO_COMMIT_CONFIG, false);
```

Set the ack-mode to MANUAL_IMMEDIATE:

```java
factory.getContainerProperties().setAckMode(AbstractMessageListenerContainer.AckMode.MANUAL_IMMEDIATE);
```

Commit the offset manually, like this:
```java
@KafkaListener(topics = "${alpha.feature.akka.channel.kafka.topic:servicecomb-pack-actor-event}")
public void listener(BaseEvent baseEvent, , Acknowledgment ack){
  try{
     actorEventSink.send(baseEvent);
     ack.acknowledge();
  }catch(){
     xx
  }   
}
```

Reply:
change auto commit mode 
change ack